lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -2599,3 +2602,31 @@ def test_course_navigation(self):
             },
             response.data
         )
+
+    def test_course_aggregate_metadata_update_on_course_published(self):
```

Comment:
These tests need to be moved to the new app as well -- api_manager orchestration layer tests should just be concerned with the endpoints, which route through to the app.